test/Gopkg.toml

```diff
@@ -51,8 +51,8 @@ required = [
 
 [[constraint]]
   name = "github.com/openshift/api"
-  version = "3.9.0"
+  branch = "release-4.1"
```

Comment:
Why not `release-4.3` to correspond more closely to what we actually use?

Reply:
If i update to `release-4.3`
and do `dep ensure` the build fails with error
```
test/vendor/github.com/openshift/api/route/v1/generated.pb.go:448:28: m.ObjectMeta.MarshalToSizedBuffer undefined (type "github.com/openshift-knative/serverless-operator/test/vendor/k8s.io/apimachinery/pkg/apis/meta/v1".ObjectMeta has no field or method MarshalToSizedBuffer)
test/vendor/github.com/openshift/api/route/v1/generated.pb.go:539:37: m.LastTransitionTime.MarshalToSizedBuffer undefined (type *"github.com/openshift-knative/serverless-operator/test/vendor/k8s.io/apimachinery/pkg/apis/meta/v1".Time has no field or method MarshalToSizedBuffer)

Reply:
meh, I don't think we care enough.